frontend/components/buttons/RevealButton/RevealButton.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import Button from "components/buttons/Button";
+import TooltipWrapper from "components/TooltipWrapper";
+
+export interface IRevealButtonProps {
+  showBoolean: boolean;
+  baseClass: string;
+  hideString: string;
+  showString: string;
```

Comment:
Minor name change: let's stick with the lingo of other components using the word "text" instead. So maybe `hideText` and `showText`.